server/api/fileOperations.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import Router from "koa-router";
+import { NotFoundError } from "../errors";
+import auth from "../middlewares/authentication";
+import { FileOperation, User, Collection, Team } from "../models";
+import policy from "../policies";
+import { presentFileOperation } from "../presenters";
+import { getSignedUrl } from "../utils/s3";
+import pagination from "./middlewares/pagination";
+
+const { authorize } = policy;
+const router = new Router();
+
+router.post("fileOperations.list", auth(), pagination(), async (ctx) => {
+  let { sort = "createdAt", direction, type } = ctx.body;
+
+  ctx.assertPresent(type, "type is required");
+
+  if (direction !== "ASC") direction = "DESC";
+
+  const user = ctx.state.user;
+
+  const where = {
+    teamId: user.teamId,
+    type,
+  };
+
+  const team = await Team.findByPk(user.teamId);
+  authorize(user, type, team);
```

Comment:
`type` is coming directly from user input here, we're allowing them to choose what authorization they like :P 

Can use `ctx.assertIn(type, ["export", "import"], ...)` to ensure it's one of those two values instead of `assertPresent`